src/Jaeger.Core/Reporters/RemoteReporter.cs

```diff
@@ -47,7 +50,7 @@ public void Report(Span span)
             try
             {
                 // It's better to drop spans, than to block here
-                added = _commandQueue.TryAdd(new AppendCommand(this, span));
+                added = _commandQueue.Post(new AppendCommand(this, span));
             }
             catch (InvalidOperationException)
```

Comment:
According to docs, .Post doesn't throw InvalidOperationException so this may now be not needed?

Reply:
@phxnsharp Maybe I should use SendAsync instead of Post?

Reply:
By my reading, we want Post since if the queue is full, we want it to drop the command. My comment is just that InvalidOperationException is no longer thrown, so the catch clause may be dead code.

Reply:
Yeah, I removed the dead code, I was just thinking because with SendAsync we postpone the adding to the queue if it is full.

Reply:
I'm not sure that makes sense, given the purpose of the code though. The idea is that it is meant to be low profile on your instrumented code. If we have failed to keep up and the queue is full, it is intentional that the packets get lost. Otherwise you'll just start filling up stack traces and Tasks instead of the queue I think.